CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # Changelog
 
+## [0.8.0] - 2023-05-10
+
+### Features
+- Add a new synchronization tool to automatically synchronize data from online storage to offline storage (#3256 @vagetablechic)
+- Support the new built-in functions `var_samp`, `var_pop`, `entropy`, `earth_distance`, `nth_value_where` and `add_months` (#3046 #3193 @aceforeverd)
+- Support batch read for openmldb-spark-connector (#3070 @tobegit3hub)
+- Support Kubernetes as a TaskManager backend for the offline engine (#3147 #3157 #3185 @tobegit3hub)
+- Support getting yarn log for `SHOW JOBLOG` (#3163 @tobegit3hub)
```

Comment:
this should be removed once the PR reverted

Reply:
ok